demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -221,9 +221,7 @@ def prepare_for_upload(
             raise FileNotFoundError(f"Could not find file {self.path}")
         data = self.data
         logger.debug(f"preparing {self.path}")
-        return MarketplaceSuffixPreparer.prepare(
-            data, current_marketplace, self.marketplaces
```

Comment:
how is this change relevant?

Reply:
This argument wasn't used and is not relevant.
We prepare the content item for a specific marketplace, and we need to provide only the marketplace we work on, and not all the marketplaces this content item supports.